terraform/acm.tf

```diff
@@ -8,10 +8,38 @@ resource "aws_acm_certificate" "ssl_certificate" {
   lifecycle {
     create_before_destroy = true
   }
+
+data "aws_route53_zone" "vickers.codes" {
```

Comment:
Terraform naming standards are to use underscores in resource names. Please update all resource names and their references to use underscores instead of dots (.) thanks!

As this resource has been applied and is being tracked in terraform state. You must issue some terraform state move commands to point the resource to the new name BEFORE applying.